src/components/WebexActivity/WebexActivity.stories.js

```diff
@@ -0,0 +1,178 @@
+import React, {Fragment} from 'react';
+import {storiesOf} from '@storybook/react';
+import {addDays, getDay, subDays} from 'date-fns';
+
+import {ActivitiesJSONAdapter, PeopleJSONAdapter} from '../../adapters';
+import {activities, people} from '../../data';
+
+import WebexActivity from './WebexActivity';
+
+// Setup for the stories
+const [activityID] = Object.keys(activities);
+const stories = storiesOf('Webex Activity', module);
+const newActivities = {};
+
+// Stories
+stories.add('default', () => (
+  <WebexActivity
+    activityID={activityID}
+    adapters={{activitiesAdapter: new ActivitiesJSONAdapter(activities), peopleAdapter: new PeopleJSONAdapter(people)}}
```

Comment:
Why isn't the style of this object broken down like the rest?

Reply:
its shorter